lib/docs.js

```diff
@@ -256,6 +257,32 @@
  * @property {object} text.width - The width of the text, including truncation and ellipsis.
  * @property {object} text.value - The value of the text, including truncation and ellipsis.
  */
+/**
+ * The additional function to invoke when a cell is filled. The return value of this
+ * function sets the new data of the cell.
+ * @function
+ * @name canvasDataGrid.fillCellCallback
+ * @param {object} args - Arguments that are passed to the function.
+ * @param {array} args.rows - Data that is being filled into all the cells, including this one.
+ * @param {string} args.direction - Direction of the fill operation, which can be 'horizontal' for x axis, 'vertical' for y axis, or 'both' when it goes both ways, e.g., when pasting unevenly.
+ * @param {object} args.newRowData - New row data that is going to replace 'existingRowData'. The result of this function will be kept in this object.
+ * @param {object} args.existingRowData - Existing data of the row. This will be replaced with 'newRowData'.
+ * @param {number} args.rowIndex - Index of the row on the grid.
+ * @param {number} args.rowPosition - Current position/index of the 'rows'. Starts with '0'.
```

Comment:
Is this `rowPosition` a relative position, relative to the start of the selection?

If so, I'd like to suggest we call this `rowOffset` — in my mind `rowPosition` implies a more absolute location, `rowOffset` indicates its relative nature. Same for `columnPosition -> columnOffset`

Reply:
The funny thing is `rowPosition` and `columnPosition` are not provided to the callback, as I already removed them, thinking they were confusing. Apparently, I also forgot to remove them in the docs. So, thanks for catching that :smile: